publisher/src/main/java/com/pavlovmedia/oss/jaxrs/publisher/impl/JaxReference.java

```diff
@@ -10,14 +9,14 @@
  *
  */
 public class JaxReference {
-    public final ServiceReference serivceReference;
+    public final ServiceReference<?> serivceReference;
```

Comment:
serviceReference was mis-spelled in the original, can we fix it?

Reply:
Yes.